src/client/clientmap.cpp

```diff
@@ -1271,13 +1267,11 @@ void ClientMap::updateDrawListShadow(v3f shadow_light_pos, v3f shadow_light_dir,
 			continue;
```

Comment:
to reduce the chance of accidentally modifying something we could change the type to `const MapSector *` above here

Reply:
I bet you could even use some C++ trickery to force the caller to use a `const` object to call this method.

Reply:
Also possible but the following trick works and is simple:
![grafik](https://github.com/minetest/minetest/assets/1042418/c9476068-761f-4038-af22-c1456b6afc45)

Reply:
Hah. I had the same thought... Implemented.